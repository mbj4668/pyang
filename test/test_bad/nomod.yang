not-module n { // error: invalid top element
}
