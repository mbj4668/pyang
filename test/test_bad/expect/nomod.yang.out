nomod.yang:1: warning: WBAD_MODULE_NAME
nomod.yang:1: error: UNEXPECTED_KEYWORD_N
